javascript/javascript1/week3/seriesDuration.js

```diff
@@ -0,0 +1,39 @@
+const seriesDurations = [
+    {
+      title: "Game of thrones",
+      days: 3,
+      hours: 1,
+      minutes: 0,
+    },
+    {
+      title: "Sopranos",
+      days: 3,
+      hours: 14,
+      minutes: 20,
+    },
+    {
+      title: "The Wire",
+      days: 2,
+      hours: 12,
+      minutes: 10,
+    },
+    {
+        title: "Teen Wolf",
+        days: 3,
+        hours: 12,
+        minutes: 40,
+    }
+  ];
+
+  function logOutSeriesText() {
+    const life = 80*365*24*60
+    let seriesSpan = 0
+    seriesDurations.forEach(a=>{
```

Comment:
Try to replace the variable name `a` with something more meaningful.

Reply:
Done